### ISSUE TITLE
LORE ONNX Export

### ISSUE THREAD

**Issue by Holmes2002** [AlibabaResearch/AdvancedLiterateMachinery#115]
Does anyone know how to export LORE ONNX? I applied the same export with CenterNet but without success.

**Comment by SWHL** [comment]
You can try to use the LORE ONNX model in the repo [TableStructureRec](https://github.com/RapidAI/TableStructureRec)

**Comment by Holmes2002** [comment]
@SWHL Hi, It is great work! However, I just want to export Lore to ONNX and I want to use my own Table Recovery due to low performance when using with large and complex table. Can you share how to convert to ONNX for me ?

**Comment by SWHL** [comment]
The LORE onnx model was converted from my previous job, and the conversion envrionment was lost at that time. 
This weekend, I wiil try to see if I can convert successfully again.

**Comment by SWHL** [comment]
@Holmes2002 hello, I reorganized the conversion code, and there is no problem in my test here. You can have a try.

This is the repo link: https://github.com/SWHL/ConvertLOREToONNX

**Comment by Holmes2002** [comment]
Hi, Thank you for your repo to convert Lore to ONNX. However, seem that you converted model ckpt_wireless-Resnet 18 to ONNX. I want to convert models DLA-34 to ONNX. Have you tried converting the DLA model to ONNX?

**Comment by SWHL** [comment]
I haven't tried DLA before.

**Comment by yangy996** [comment]
nice

**Comment by phamkhactu** [comment]
@Holmes2002 

Have you had any information for DLA-34(onnx)?

**Comment by Holmes2002** [comment]
@phamkhactu No, I successfully converted DLA-34 to ONNX but the inference doesn't work and I don't understand why

**Comment by phamkhactu** [comment]
> @phamkhactu No, I successfully converted DLA-34 to ONNX but the inference doesn't work and I don't understand why

Could you give me snip code for converting onnx if convenience? Or you can compare your onnx with [onnx themself](https://github.com/RapidAI/TableStructureRec/releases/tag/v0.0.0). I don't know it extractly which is  by a author.

**Comment by Holmes2002** [comment]
@phamkhactu Hi, this is code for converting to ONNX in final file 
```
https://github.com/Holmes2002/Table-Recognition/blob/main/LORE-TSR/src/lib/models/networks/pose_dla_dcn.py
```
My own inference code but did't work
```
https://github.com/Holmes2002/Table-Recognition/blob/main/LORE-TSR/src/lib/models/networks/inference_onnx.py
```
You can check and modify code. Announce to me if you success
